stl/inc/format

```diff
@@ -69,14 +69,6 @@ struct _Auto_id_tag {};
 // clang-format off
 template <class _Ty, class _CharT>
 concept _Parse_spec_callbacks = requires(_Ty _At, basic_string_view<_CharT> _Sv, _Align _Aln, _Sign _Sgn) {
-    { _At._On_align(_Aln) } -> same_as<void>;
-    { _At._On_fill(_Sv) } -> same_as<void>;
-    { _At._On_width(int{}) } -> same_as<void>;
-    { _At._On_dynamic_width(size_t{}) } -> same_as<void>;
-    { _At._On_dynamic_width(_Auto_id_tag{}) } -> same_as<void>;
-    { _At._On_precision(int{}) } -> same_as<void>;
-    { _At._On_dynamic_precision(size_t{}) } -> same_as<void>;
-    { _At._On_dynamic_precision(_Auto_id_tag{}) } -> same_as<void>;

```

Comment:
you need to include the other concepts, not just remove their requirements. So like

```c++
concept _Parse_spec_callbacks = _Parse_align_callbacks<_Ty, _CharT> && _Parse_bla_callbacks<_Ty, _CharT> && requires(...) { ... }
```

Reply:
Ah gotcha, thanks! I'll do the same for the `_Chrono_parse_spec_callbacks`.  I manually formatted it as best I could, since this region is `clang-format off`.